lib/vector-core/Cargo.toml

```diff
@@ -81,7 +81,9 @@ value = { path = "../value", default-features = false, features = ["lua", "toml"
 api = ["async-graphql", "value/api"]
 default = []
 lua = ["mlua", "tokio-stream"]
-vrl = ["vrl-core", "enrichment"]
+# Rename to "vrl" once we use a release with stable `-Z namespaced-features`:
+# https://doc.rust-lang.org/cargo/reference/unstable.html#namespaced-features
+vrl_ = ["vrl", "enrichment"]
```

Comment:
Alternatively, import vrl as a different name (e.g. `vrl_lib`) so you can keep the `vrl` feature name here:

```toml
 vrl_lib = { package = "vrl", path = "../vrl/vrl", optional = true }
```

Then, in `src/lib.rs` in vector-core, you can potentially do `use vrl_lib as vrl`?

Reply:
Yeah, I thought about this alternative as well. However, that would only work with prefixes like `crate::` or `super::`, so I considered both to be more of a temporary workaround. I agree that it's a bit less dirty than adding a `_`, though.

Given that the PR to stabilize this feature has already been merged https://github.com/rust-lang/cargo/pull/10269 and released with Cargo `1.60.0` we should be able to move off of any workaround pretty soon.